docs/reference/modules/node.asciidoc

```diff
@@ -22,6 +22,17 @@ TIP: As the cluster grows and in particular if you have large {ml} jobs or
 {ctransforms}, consider separating dedicated master-eligible nodes from
 dedicated data nodes, {ml} nodes, and {transform} nodes.
 
+[[node-roles]]
+==== Nore roles
+
+You can define the roles of a node by setting `node.roles`. If you don't 
+configure this setting, then the node has the following roles by default: 
+`master`, `data`, `ingest`, `transform`, `ml`.
+
+If you use the `node.roles` setting, then all required roles must be explicitly 
+set as the values that you provide for `node.roles` overwrite the default 
+behavior.
```

Comment:
This sentence was a bit long and hard to follow. If I understood it correctly, I think you can simplify it to something like: 
> If you set `node.roles`, the node is assigned only the roles you specify.